packages/flutter/lib/src/material/bottom_sheet.dart

```diff
@@ -574,50 +573,32 @@ class _RenderBottomSheetLayoutWithSizeListener extends RenderShiftedBox {
     markNeedsLayout();
   }
 
-  Size _getSize(BoxConstraints constraints) {
-    return constraints.constrain(constraints.biggest);
-  }
-
   @override
   double computeMinIntrinsicWidth(double height) {
-    final double width = _getSize(BoxConstraints.tightForFinite(height: height)).width;
-    if (width.isFinite) {
-      return width;
-    }
-    return 0.0;
+    final double width = BoxConstraints.tightForFinite(height: height).biggest.width;
+    return width.isFinite ? width : 0.0;
   }
```

Comment:
These methods are looking _much_ better now, thanks!

Now that it's been cleaned up a bit, I'm wondering: is it at all possible for this to return anything other than zero? (It seems like `width` will always be set to `double.infinity`, thus `0.0` is always returned.)

Reply:
Oh that's a good catch! It does seem like most of the compute intrinsic methods would always return 0 regardless.